site/index.md

```diff
@@ -9,9 +9,11 @@ layout: home
 #  a makerspace in Roanoke, VA.
 ---
 
+### MAKE Roanoke is launching a Kickstarter campaign to open a Makerspace in Roanoke, VA
+
 MAKE Roanoke has already raised nearly $10,000 from individual contributions, membership dues, class fees and more. We’re currently negotiating a lease for a location in the heart of Roanoke. We've also received some generous donations of equipment, but we need YOUR HELP to cross the finish line!
 
-![](/assets/images/Kickstarter.png)
+![https://meetup.com/make-roanoke](/assets/images/20241017_launch_party.png)
```

Comment:
```suggestion
```

Reply:
We are using the kickstarter launch party image

https://discord.com/channels/1179210982258647100/1287907844695396385/1296583231554060348